Tasks/AzureResourceGroupDeployment/task.json

```diff
@@ -380,6 +380,7 @@
         "OnlyTokenAuthAllowed": "Endpoint can be of Token authorization type only.",
         "DeploymentGroupEndpointUrlCannotBeEmpty": "Deployment group endpoint url cannot be empty",
         "DeploymentGroupEndpointPatTokenCannotBeEmpty": "Deployment group endpoint personal access token cannot be empty",
-        "ErrorWhileParsingParameter": "There was an error while overriding '%s' parameter because of '%s'."
+        "ErrorWhileParsingParameter": "There was an error while overriding '%s' parameter because of '%s'.",
+        "ResourceNameCannotBeNull": "Resource name cannot be null."
```

Comment:
this is used in common azure-arm-rest. currently, common modules' resource strings are also loaded from task.json.